assets/src/application/services/theme/theme.ts

```diff
@@ -2,8 +2,22 @@ import { theme as chakraTheme, DefaultTheme } from '@chakra-ui/core';
 
 type CustomTheme = DefaultTheme;
 
+const breakpoints = ['30em', '48.875em', '62em', '80em'];
```

Comment:
![Peek 2020-05-18 13-55](https://user-images.githubusercontent.com/1477453/82205450-4cc8ab00-990f-11ea-8cd8-bd2b5a3ef6f0.gif)

Reply:
the `em` values of `48.875em` was converted here: 

![image](https://user-images.githubusercontent.com/1477453/82205492-59e59a00-990f-11ea-8c32-95eae718974d.png)

Reply:
http://pxtoem.com/